etc/lmod-setup.sh

```diff
@@ -20,7 +20,7 @@ elif [ "$L_MACHINE" = singularity ]; then
    module purge
 
 elif [ "$L_MACHINE" = gaea ]; then
-   export BASH_ENV="/lustre/f2/pdata/esrl/gsd/contrib/lua-5.1.4.9/init/init_lmod.sh"
+   export BASH_ENV="/lustre/f2/pdata/esrl/gsd/contrib/lua-5.1.4.9/lmod/lmod/init/bash"
```

Comment:
Could you do similar modification in `etc/lmod-setup.csh` for those who use tcsh login?

Reply:
Also, now that the modulefiles that the cray module app seem to be unloaded instead of re-loaded, which the previous `init_lmod.sh` did, maybe it is  appropriate to add `module purge` to it just like macos and singularity do?